### ISSUE TITLE
Implement Symfony handlers for Auth0 custom db connection

### ISSUE THREAD

**Issue by crucialfelix** [Plant-for-the-Planet-org/treecounter-app#1291]
with BasicAuth?

**Comment by jmiridis** [comment]
@crucialfelix 
I have created a connection to my local database and played around with the the default Node.js script for MySQL. I can retrieve id, email, and password but the encryption used in our project (`sha512`) does not match the encryption used with auth0 (`bcrypt`), so password validation fails. Not sure how to proceed.

You are mentioning POST handlers. Not sure what you mean with that. I assume you mean some kind of API that would allow incoming POSTs from Auth0? What would the posted data and the expected response be?

**Comment by jmiridis** [comment]
In case we need to add some non-standard npm module to handle the password validation, teh documentation recommends making a request in the Auth0 support portal:
https://support.auth0.com/

see https://auth0.com/docs/connections/database/custom-db/overview-custom-db-connections#npm-modules

**Comment by crucialfelix** [comment]
Yes, I meant a POST api. I think this is far preferable to exposing MySQL to the internet (even though you could set up firewall rules).

We write the little node script for their side, so we can determine the exact shape of the request.

Password encryption is still an issue though. That can be solved though. ( https://github.com/Plant-for-the-Planet-org/treecounter-app/issues/1299 )

**Comment by crucialfelix** [comment]
I've activated custom database hooks for these two tenants/domains:

- TTC Devel (https://devel.trilliontreecampaign.org/app_dev.php/auth0-admin/)
- TTC Test (https://test.trilliontreecampaign.org/app_dev.php/auth0-admin/)

No BasicAuth yet

All the handlers described above are now active. There is a test button on Auth0 custom databases to make it easy to test your work in progress.
